### PR TITLE
According to the Slack docs we need to escape <>& chars

### PR DESCRIPTION
**Pull request by ChrisBr**
https://api.slack.com/docs/message-formatting

See e.g. https://ckpd.slack.com/archives/C8GQ6ND3P/p1559201762001000

### CODE REVIEW

File: lib/issue_title.rb [hunk 1]
```diff
@@ -4,12 +4,22 @@ class IssueTitle < SimpleDelegator
...
+  }.freeze
+
   def tags
     raw_tags.map do |tag|
       VALID_TAGS[tag]
     end.compact
   end
 
+  def to_s
+    self.gsub(/[#{ESCAPED_CHARACTERS.keys.join}]/, ESCAPED_CHARACTERS)
```
Comment (balvig) [inline lib/issue_title.rb]: Could we replace this with

```suggestion
    CGI.escapeHTML(self)
```

File: test/issue_title_test.rb [hunk 2]
```diff
@@ -6,4 +6,10 @@ def test_tags
 
     assert_equal [:wip], title.tags
   end
+
+  def test_title_escaping
+    title = IssueTitle.new("With <div>HTML</div> & ampersamp")
```
Comment (davidstosik) [inline test/issue_title_test.rb]: Nit
```suggestion
    title = IssueTitle.new("With <div>HTML</div> & ampersand")
```

File: lib/issue_title.rb (hunk 1 above)
Comment (davidstosik) [reply lib/issue_title.rb]: :+1: Escaping more than required should be harmless.

Comment (ChrisBr) [reply lib/issue_title.rb]: I was already using `CGI.escape`, however, according to the Slack API docu: 

> There are three characters you must convert to HTML entities and only three: &, <, and. Don't HTML entity-encode the entire message.

So I think in this case, escaping more than required is not harmless 🤔

**Comment by ChrisBr** [comment]
♻️

File: lib/issue_title.rb (hunk 1 above)
Comment (balvig) [reply lib/issue_title.rb]: 👍

File: test/issue_title_test.rb [hunk 3]
```diff
@@ -6,4 +6,10 @@ def test_tags
 
     assert_equal [:wip], title.tags
   end
+
+  def test_title_escaping
+    title = IssueTitle.new("With <div>HTML</div> & ampersand")
+
+    assert_equal title.to_s, "With &lt;div&gt;HTML&lt;/div&gt; &amp; ampersamp"
```
Comment (ChrisBr) [inline test/issue_title_test.rb]:
```suggestion
    assert_equal title.to_s, "With &lt;div&gt;HTML&lt;/div&gt; &amp; ampersand"
```

**Pull request merged**